### PR TITLE
Fixed #32585 -- Fixed Value(output_field=DecimalField()) crash on SQLite.

### PR DESCRIPTION
**Pull request by Adrian-at-CrimsonAuzre**
Super simple fix, don't know if it is the best way but it works.

Fixes ticket [#32585](https://code.djangoproject.com/ticket/32585)

### CODE REVIEW

File: django/db/backends/sqlite3/operations.py [hunk 1]
```diff
@@ -314,6 +314,8 @@ def converter(value, expression, connection):
         else:
             def converter(value, expression, connection):
                 if value is not None:
+                    if isinstance(value, str):
+                        return create_decimal(float(value))
```
Comment (felixxm) [inline django/db/backends/sqlite3/operations.py:318]: I think we should make `Value` inherit from `SQLiteNumericMixin`.

**Review by felixxm — COMMENTED** [review]
@Adrian-at-CrimsonAuzre Thanks for this patch :+1: 

Tests are required.

File: django/db/backends/sqlite3/operations.py (hunk 1 above)
Comment (Adrian-at-CrimsonAuzre) [reply django/db/backends/sqlite3/operations.py:318]: I just patched it so I could get my testDB code working, don't really know enough about the inner workings of Django to do a proper patch. If someone else wants to do it properly then go for it.

Comment (felixxm) [reply django/db/backends/sqlite3/operations.py:318]: OK, thanks. Don't worry someone will prepare a patch.

**Pull request closed without merging**